react-app/src/app/components/RouteHandler/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
+
+import Game from '../../screens/Game';
+import Login from '../../screens/Login';
+
+function App() {
+  return (
+    <Router>
+      <Fragment>
+        <Route exact path="/" render={() => <Redirect to="/login" />} />
+        <Route path="/game" component={Game} />
+        <Route path="/login" component={Login} />
```

Comment:
Instead of having the login logic in all the components. How about creating a `PrivateRoute` component which checks for the `loggedIn` if it is logged then it will redirect the user to the desire page, if not it will go to the login.

`<PrivateRoute path="/game" component={Game} />`